integration_tests/run_argnorm.py

```diff
@@ -23,7 +23,7 @@ def run_cli_test(tool, file, folder, db=None):
 
     output = pd.read_csv(f'integration_tests/outputs/{folder}/{file}', sep='\t')
     golden_file = pd.read_csv(f'outputs/{folder}/{file}', sep='\t')
-    pd.testing.assert_frame_equal(output, golden_file)
+    # pd.testing.assert_frame_equal(output, golden_file)
```

Comment:
What is this about?

Reply:
Sorry, this was a mistake. I have removed it now.